src/lang/english.txt

```diff
@@ -1584,8 +1584,11 @@ STR_CONFIG_SETTING_DRAG_SIGNALS_FIXED_DISTANCE                  :When dragging,
 STR_CONFIG_SETTING_DRAG_SIGNALS_FIXED_DISTANCE_HELPTEXT         :Select the behaviour of signal placement when Ctrl+dragging signals. If disabled, signals are placed around tunnels or bridges to avoid long stretches without signals. If enabled, signals are placed every n tiles, making alignment of signals at parallel tracks easier
 STR_CONFIG_SETTING_SEMAPHORE_BUILD_BEFORE_DATE                  :Automatically build semaphores before: {STRING2}
 STR_CONFIG_SETTING_SEMAPHORE_BUILD_BEFORE_DATE_HELPTEXT         :Set the year when electric signals will be used for tracks. Before this year, non-electric signals will be used (which have the exact same function, but different looks)
-STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI                            :Enable the signal GUI: {STRING2}
-STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI_HELPTEXT                   :Display a window for choosing signal types to build, instead of only window-less signal-type rotation with Ctrl+clicking on built signals
+STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI                            :Show signal GUI: {STRING2}
+STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI_HELPTEXT                   :Display a window for choosing signal types to build. If None, signal types can be cycled by Ctrl+clicking on built signals
+STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI_PATH                       :Path signals only {BLUE}(Recommended)
+STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI_ALL                        :All signals
+STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI_NONE                       :None
```

Comment:
We now have 3 settings with nearly identical meaning. This feels like we are overcomplicating things.

You can now have the GUI show only Path signals, but have the default signal type set to block signals, while cycling through all.

Dunno what you could do about it, but brrrrr, it is becoming a bit weird-ish now :D

Reply:
Haha, it seems #7504 also came to a similar conclusion. How does your solution differ from #7504? To me it seems to be the same solution with slightly different implementation?

Reply:
This is based on #7504, yes. In that, PeterN suggested using planes to hide the widgets instead, but nielsmh ran into problems with this and gave up. I saw JGR successfully using planes to hide signals in his recent realistic braking patch, and decided to finish the job.

Reply:
Ah :D I read the PR as: this changing the GUI is not going to work, the GUI needs a redesign :P Hence my question :D

That does leave open my question if we really want 3 settings doing similar but different things .. and how one should interact with the others, of course :)